src/App.js

```diff
@@ -32,7 +40,7 @@ class App extends React.Component {
         <h1>🍅 Rancid Tomatillos 🍅</h1>
         <Header />
         {!this.state.isSingleMovie && <Movies movies={this.state.allMovies} displayMovieInfo={this.displayMovieInfo}/>}
-        {this.state.isSingleMovie && <SingleMovieCard movie={this.state.singleMovie} displayMovieInfo={this.displayMovieInfo} hideSingleView={this.hideSingleView}singleMovie={this.state.singleMovie}/>}
+        {this.state.isSingleMovie && <SingleMovieCard  displayMovieInfo={this.displayMovieInfo} hideSingleView={this.hideSingleView} singleMovie={this.state.singleMovie} />}
```

Comment:
To improve readability, you can move each of the props to a new line. Otherwise, this is getting really long with all of the props.